assets/dns/daemonset.yaml

```diff
@@ -77,7 +77,7 @@ spec:
           while true; do
             declare -A svc_ips
             for svc in "${services[@]}"; do
-              # Fetch service IP from cluster dns if present
+              # Fetch service IP from dns if present
```

Comment:
It makes sense to say "cluster" DNS here because the code in question is resolving a cluster-internal name using the cluster-internal nameserver.

Reply:
fixed